modules/apps/callbacks/testing/simapp/simd/cmd/root.go

```diff
@@ -257,7 +257,7 @@ func txCommand() *cobra.Command {
 		authcmd.GetBroadcastCommand(),
 		authcmd.GetEncodeCommand(),
 		authcmd.GetDecodeCommand(),
-		authcmd.GetAuxToFeeCommand(),
```

Comment:
This is [not available anymore](https://github.com/cosmos/cosmos-sdk/blame/v0.50.0-rc.1/simapp/simd/cmd/commands.go#L182-L192).